lib/core/api.js

```diff
@@ -728,6 +734,9 @@ api.errors.log = function(error, message, properties) {
     } else {
       rollbar.error(error);
     }
+    if(error.originalError){
+      rollbar.error(error.originalError);
```

Comment:
This means there will be multiple error reports for the same event, right? This is already happening to some extent, so wouldn't be too bad, but I'm wondering if we should rather just log the original error when available, adding the new error object as a custom field (which will then show up under `message.extra`). As an example, see line 733 where the blob ID is added as a custom field.